module/Application/view/application/index/index.phtml

```diff
@@ -1,43 +1,43 @@
 <div class="jumbotron">
-    <h1><?php echo sprintf($this->translate('Welcome to %sZend Framework 2%s'), '<span class="zf-green">', '</span>') ?></h1>
-    <p><?php echo sprintf($this->translate('Congratulations! You have successfully installed the %sZF2 Skeleton Application%s. You are currently running Zend Framework version %s. This skeleton can serve as a simple starting point for you to begin building your application on ZF2.'), '<a href="https://github.com/zendframework/ZendSkeletonApplication" target="_blank">', '</a>', \Zend\Version\Version::VERSION) ?></p>
-    <p><a class="btn btn-success btn-lg" href="https://github.com/zendframework/zf2" target="_blank"><?php echo $this->translate('Fork Zend Framework 2 on GitHub') ?> &raquo;</a></p>
+    <h1><?php echo sprintf('Welcome to %sZend Framework%s', '<span class="zf-green">', '</span>') ?></h1>
```

Comment:
These view scripts were really horrible to read. Can we simplify them a bit? All this `sprintf()` feels really unnecessary